source/_integrations/insteon.markdown

```diff
@@ -81,101 +81,86 @@ insteon:
-      required: false
-      type: string
-    product_key:
-      description: are more advanced options and will typically not be used.
-      required: false
-      type: integer
+description: Override the default device definition.
+required: false
+type: list
+keys:
+address:
+description: is found on the device itself in the form 1A.2B.3C or 1a2b3c.
+required: true
+type: string
+cat:
+description: is found in the back of the device's User Guide in the form of 0x00 - 0xff.
+required: false
+type: integer
+subcat:
+description: is found in the back of the device's User Guide in the form of 0x00 - 0xff.
+required: false
+type: integer
+firmware:
+description: are more advanced options and will typically not be used.
+required: false
+type: string
+product_key:
+description: are more advanced options and will typically not be used.
+required: false
+type: integer
 x10_devices:
-  description: Define X10 devices to control or respond to.
-  required: false
-  type: list
-  keys:
-    housecode:
-      description: is the X10 housecode values a - p
-      required: true
-      type: string
-    unitcode:
-      description: is the X10 unit code values 1 - 16
-      required: true
-      type: integer
-    platform:
-      description: "is the Home Assistant Platform to associate the device with. The following platforms are supported: binary_sensor: Used for on/off devices or keypad buttons that are read-only. light: Used for dimmable X10 devices. switch: Used for On/Off X10 devices."
-      required: true
-      type: string
-    dim_steps:
-      description: is the number of dim/bright steps the device supports. Used for dimmable X10 devices only.
-      required: false
-      default: 22
-      type: integer
-x10_all_units_off:
-  description: Creates a binary_sensor that responds to the X10 standard command for All Units Off.
-  required: false
-  type: string
-x10_all_lights_on:
-  description: Creates a binary_sensor that responds to the X10 standard command for All Lights On
-  required: false
-  type: string
-x10_all_lights_off:
-  description: Creates a binary_sensor that responds to the X10 standard command for All Lights Off
-  required: false
-  type: string
+description: Define X10 devices to control or respond to.
+required: false
+type: list
+keys:
+housecode:
+description: is the X10 housecode values a - p
+required: true
+type: string
+unitcode:
+description: is the X10 unit code values 1 - 16
+required: true
+type: integer
+platform:
+description: "is the Home Assistant Platform to associate the device with. The following platforms are supported: binary_sensor: Used for on/off devices or keypad buttons that are read-only. light: Used for dimmable X10 devices. switch: Used for On/Off X10 devices."
+required: true
+type: string
+dim_steps:
+description: is the number of dim/bright steps the device supports. Used for dimmable X10 devices only.
+required: false
+default: 22
+type: integer
```

Comment:
Until here

Reply:
@klaasnicolaas I made the change above but I am a bit confused. When I look at the changes they appear correct in the editor but they don't appear to be correct after saving. Can you have a look and tell me if they are correct?